tests/framework/src/runner.rs

```diff
@@ -574,3 +574,46 @@ fn assert_text_matches(ruffle: &str, flash: &str) -> Result<()> {
         Ok(())
     }
 }
+
+fn automated_key_to_key_code(automated_key: AutomatedKey) -> KeyCode {
+    match automated_key {
+        AutomatedKey::Char(ch) => KeyCode::from_code(ch.to_ascii_uppercase() as u32),
```

Comment:
I think we should assert that the char is something like a..z.

Reply:
Actually key codes may represent non-ASCII characters. If you try inputting a non-ASCII character you'll get its Unicode code point as the value of key code.

That's because key codes are logical in Flash, not physical.